common/gpu/amd/default.nix

```diff
@@ -1,23 +1,35 @@
 { config, lib, pkgs, ... }:
 
 {
-  boot.initrd.kernelModules = [ "amdgpu" ];
-  services.xserver.videoDrivers = [ "amdgpu" ];
+  options.hardware.amdgpu.loadAtStage1 = lib.mkEnableOption (lib.mdDoc 
+    "Load `amdgpu` kernelModule at stage 1. (Add `amdgpu` to `boot.initrd.kernelModules`)"
```

Comment:
the description templates `Whether to enable ${description}.`, so it should (1) start with appropriate casing (likely lower-case) and (2) continue the sentence

```suggestion
    "loading `amdgpu` kernelModule at stage 1. (Add `amdgpu` to `boot.initrd.kernelModules`)"
```